src/app/extensions/quoting/shared/product/product-add-to-quote-dialog/product-add-to-quote-dialog.component.ts

```diff
@@ -36,6 +41,8 @@ export class ProductAddToQuoteDialogComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
+    this.user$ = this.accountFacade.user$;
+    this.selectedQuoteRequest$ = this.quotingFacade.quoteRequest$;
     this.activeQuoteRequest$ = this.quotingFacade.activeQuoteRequest$;
```

Comment:
The `selectedQuoteRequest$` and `activeQuoteRequest$` handling does not work as expected if there is more than one 'NEW' unsubmitted quote request. In that case submitting the quote request in the dialog will not present the "Thank you for your quote" message but switch to the now different active quote request.

The whole handling/distinction of the `selectedQuoteRequest$` and `activeQuoteRequest$` needs to be more clear here (maybe some comments in the code?). Maybe there should not be both within the dialog?